Release AD7134

jansunil opened the pull request:
Release AD7134 project with the STM32H563 support

ribdp commented on projects/ad7134_iio/app/app_config.c:
```diff
@@ -172,7 +170,7 @@ static struct eeprom_24xx32a_init_param eeprom_extra_init_params = {
 /* EEPROM init parameters */
 static struct no_os_eeprom_init_param eeprom_init_params = {
-	.device_id = 0,
+	.device_id = I2C_DEVICE_ID,
```
The macro would also need to be added in app_config_mbed.h I think?

jansunil replied:
It is already present in the mbed config-https://github.com/analogdevicesinc/precision-converters-firmware/blob/release-ad777x-h563/projects/ad7134_iio/app/app_config_mbed.h#L85

ribdp replied:
My bad, missed that somehow

ribdp commented on projects/ad7134_iio/app/app_config_stm32.c:
```diff
@@ -3,7 +3,7 @@
  * @details This module contains the STM32 platform specific configurations
 ********************************************************************************
-* Copyright (c) 2021 Analog Devices, Inc.
+* Copyright (c) 2021,2023 Analog Devices, Inc.
```
Should this be 2024?

jansunil replied:
Updated the copyrights

D-Disha commented:
Can the second commit message also include a mention of the tests folder update

SaikiranGudla commented on libraries/no-OS.lib:
```diff
@@ -1 +1 @@
-https://github.com/analogdevicesinc/no-OS/#926e22ab5b1309d62a25873b8b07bf581e674c32
\ No newline at end of file
+hhttps://github.com/analogdevicesinc/no-OS/#68bf422d83850074e3d721e7e3cc6b67e22bc433
```
Correction: `https`

jansunil replied:
Updated the link

jansunil commented:
changelog:

Updated the libraries/.mbedignore to exclude the new ad796x drivers added in the no-OS repo
Updated the .extsettings file to include the no_os_dma.c file
